__tests__/cpu_test.re

```diff
@@ -67,9 +67,18 @@ describe("CPU", () => {
     );
   });
 
+  describe("now", () => {
+    let cpu = Cpu.copy(initial);
+    cpu.pc = 0xc000;
+    let actual = cpu |> Cpu.now |> String.trim;
```

Comment:
`nb` I've tended to write `fn(arg) |> ...` instead of `arg |> fn |> ...`. Do you have a general preference for one or the other?

Reply:
No strong preference. I also see the docs are pushing towards the style you used elsewhere. I'll try and find some time to play around with the different piping options/techniques and we can discuss on our next call.